README.txt

```diff
@@ -3,7 +3,7 @@ Build a docker image:
   # docker build -t ostree-upload-server .
 
 
-Check configuration in remotes.conf and flatpak-import.conf
+Check configuration in remotes.conf and bundle-import.conf
```

Comment:
This should be changed back now that it's still using `flatpak-import.conf`.

Reply:
Will fix